governance/polls/Vault Compensation Denomination - April 13, 2020.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Vault Compensation Denomination Poll - April 13, 2020
+summary: Signal whether to provide compensation to vault holders in the vault's token (ETH/BAT) or in DAI
+discussion_link: https://forum.makerdao.com/t/1822
+poll_rules: The voter may select to vote for one of the poll options or they may elect to abstain from the poll entirely
+options:
+   0: Abstain
+   1: ETH or BAT (the asset that was liquidated) 
+   2: DAI
+
+---
+# Vault Compensation Denomination Poll - April 13, 2020
+
+The Maker Foundation Interim Governance Facilitator has placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the community which the community can use to signal whether to move ahead with a plan to provide compensation to vault holders.
```

Comment:
The decision whether to move ahead with compensation would already have been made?
```suggestion
The Maker Foundation Interim Governance Facilitator has placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the community which the community can use to signal whether to provide compensation to vault holders in the vault's token (ETH/BAT) or in DAI
```